src/index.js

```diff
@@ -121,3 +121,15 @@ enterIcon.addEventListener('keypress', (event) => {
 
 renderTasks();
 saveTasks(tasks);
+
+const clearButton = document.getElementById('clear-completed');
+
+clearButton.addEventListener('click', () => {
+  tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+  tasks = tasks.filter((task) => !task.completed);
+
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+
+  renderTasks();
```

Comment:
- [ ] **Optional:** When an item is deleted the index is updated, the same functionality could be implemented with the clear all completed button in your free time.